### ISSUE TITLE
[Bug]  Actualize lock files in a multi instance environment share the same path

### ISSUE THREAD

**Issue by kdanni** [FreshRSS/FreshRSS#6370]
### Describe the bug

Current installed version 1.23.1

In multi installation setup on hosted server where I have little agency instances running actualize cron jobs at same time share actualize lock files. Causing failed actualize job for every other instance after the first. 

app/actualize_script.php:
`$mutexFile = TMP_PATH . '/actualize.freshrss.lock';`

I modified the "TMP_PATH . '/actualize.freshrss.lock';`" line to unique filenames per instance. And it solved my issue for now. However, I think this issue can cause problem for others.

### To Reproduce

1. Setup multiple instance.
2. Setup cron actualize jobs to the same date. 
3. Check cron logs.
4. See error in one: "FreshRSS feeds actualization was already running, so aborting new run..."

### Expected behavior

Instances in the same environment installed different directory have different lock files.

### FreshRSS version

Current installed version 1.23.1

### Environment information

Installed on LAMP hosting service managed by Cpanel.
- PHP 7.4+

**Comment by Alkarex** [comment]
Hello,
I would suggest you make a file `./constants.local.php` with a unique temp directory for your various instances, like:

```php
<?php
define('TMP_PATH', '/tmp/freshrss1/');
```

**Comment by kdanni** [comment]
Thanks!

`constants.local.php` is a clean work around. 
Scripting the directory creation was made things scalable. Also it works before and after version upgrade.

I still think lock files names including unique part per instance prevent collision out of box. It's in a tmp folder anyway.

```php
<?php

$sysTmpDir = sys_get_temp_dir();
$dirMd5 = md5(__DIR__);

$tmpPath = $sysTmpDir . '/' . $dirMd5;
$dataPath = __DIR__ . '/data/' . $dirMd5; // Reference for finding the md5 hashed dir name.

if (!is_dir($tmpPath))
{
    mkdir($tmpPath, 0755, true);
}
if (!is_dir($dataPath))
{
    mkdir($dataPath, 0755, true);
}

define('TMP_PATH', $tmpPath);
```

**Comment by Alkarex** [comment]
> I still think lock files names including unique part per instance prevent collision out of box

Yes, that would be reasonable. We need to find what value to use to be unique there. Maybe our salt.

**Comment by Frenzie** [comment]
> Maybe our salt.

That sounds leaky.

**Comment by Alkarex** [comment]
I ment based on 🤓

**Comment by Alkarex** [comment]
Tests welcome: https://github.com/FreshRSS/FreshRSS/pull/9045
